packages/cactus-plugin-keychain-azure-kv/src/test/typescript/unit/api-surface.ts

```diff
@@ -0,0 +1,7 @@
+import test, { Test } from "tape";
```

Comment:
@jagpreetsinghsasan Please add a `.test` suffix to the file name (a new-ish convention)